lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -538,6 +539,11 @@ def add_resources_bundle_targets(file_accessors)
 
                 remove_pod_target_xcconfig_overrides_from_target(target.build_settings_for_spec(file_accessor.spec), resource_bundle_target)
```

Comment:
@segiddins this line continues to be a problem in which a bunch of settings _including_ `<PLATFORM>_DEPLOYMENT_TARGET` gets removed.

Reply:
I had to move `resource_bundle_target.deployment_target = deployment_target` below that line for things to function.

Reply:
Well.....kind of nevermind this is a different issue where app specs xcconfigs are merged into the resource bundles and our Watermelon podspec includes `IPHONEOS_DEPLOYMENT_TARGET` which is removed by the resource bundle target.